### DISCUSSION TITLE
return {"detail":"Hmm...something seems to have gone wrong. Maybe try me again in a little bit."}

### DISCUSSION THREAD

**Discussion by yhuso** [acheong08/ChatGPT#897] [Q&A]
I am more familiar with Java, so I am trying to rewrite the author's Python code using Java.
step 1 : I got the correct access token.
step 2: when i call 'https://chatgpt.duti.tech/api/conversation' , it shows the error message.

Is it limited by chatgpt server ?
Does anyone have any ideas？
@acheong08  
![image](https://user-images.githubusercontent.com/6768065/221358687-2351395a-b768-42b3-be81-5bdc24246f10.png)

**Comment by acheong08** [comment]
Happens with malformed requests

**Reply by yhuso** [reply]
> Happens with malformed requests

Indeed, it was due to a parameter passing error in my code.
